pack/TFM.json

```diff
@@ -0,0 +1,101 @@
+[
+    {
+        "code": "08085",
+        "cost": 5,
+        "deck_limit": 3,
+        "designer": "Card design by 2011 Joust World Champion, Brett Zeiler.",
+        "faction_code": "thenightswatch",
+        "illustrator": "",
```

Comment:
illustrator is "Diego Gisbert Llorens", based off the original CCG version of the card with the same artwork